static/js/notifications.js

```diff
@@ -217,14 +217,19 @@ exports.window_has_focus = function () {
     return window_has_focus;
 };
 
-function in_browser_notify(message, title, content) {
+function in_browser_notify(message, title, content, raw_operators, opts) {
     var notification_html = $(templates.render('notification', {gravatar_url: ui.small_avatar_url(message),
                                                                 title: title,
                                                                 content: content}));
     $('.top-right').notify({
         message: {html: notification_html},
         fadeOut: {enabled: true, delay: 4000}
     }).show();
+    $('.top-right').on('click', function () {
+        ui.change_tab_to('#home');
+        narrow.activate(raw_operators, opts);
+    });
+    setTimeout(function () {$('.top-right').unbind("click");}, 5000);
```

Comment:
@timabbott While trying to fix this issue, one of the problems I stumbled upon was that, even after a notification had faded out, you would still be able to click it. So basically, without the `setTimeout`, if you got a total of 100 notifications spread out over an entire hour while being on a Zulip server, pressing one notification would start the `narrow.activate` 100 times, causing a lot of lag. The easiest solution I came up with was disabling the clickability of a notification just after the notification has faded out using a `setTimeout`. My first version instantly got merged without anyone talking about the `setTimeout`, but there might be a better solution though. If you know a better solution, please tell me.